apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/ScalarType.kt

```diff
@@ -1,17 +1,17 @@
 package com.apollographql.apollo.api
 
 /**
- * Represents a custom GraphQL scalar type
+ * Represents a mapping from a custom GraphQL scalar type to a Java/Kotlin class
  */
 interface ScalarType {
```

Comment:
nit: should we rename this to `CustomScalarType`?

Reply:
nvm I see it's done in next PR https://github.com/apollographql/apollo-android/pull/2820